integration/audio-spectrograms/run.py

```diff
@@ -37,6 +37,9 @@
 for logger in loggers_to_mute:
     logging.getLogger(logger).setLevel(logging.ERROR)
 logging.getLogger("klio").setLevel(logging.DEBUG)
+# The concurrency logs may be different for every machine, so let's
+# just turn them off
+logging.getLogger("klio.concurrency").setLevel(logging.ERROR)
```

Comment:
FYI I didn't apply this line to _all_ integration tests. The ones I didn't apply them to don't actually look at the job output logs at all.